app/src/main/res/layout/activity_selection_control.xml

```diff
@@ -0,0 +1,120 @@
+            <TextView
+                android:id="@+id/text_view_span_usage"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="Usage"
+                tools:ignore="HardcodedText" />
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:layout_height="match_parent"
+                android:layout_width="match_parent" />
+                <TextView
+                    android:id="@+id/text_view_span_switch"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:text="Switches"
+                    tools:ignore="HardcodedText" />
+                <com.google.android.material.switchmaterial.SwitchMaterial
+                    android:id="@+id/switch_one"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:text="@string/switch_text"
+                    android:textOn="@string/text_on"
+                    android:textOff="@string/text_off"
+                    app:useMaterialThemeColors="true"
+                    android:checked="false"/>
+                <TextView
+                    android:id="@+id/text_view_span_checkbox"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:text="Checkbox"
+                    tools:ignore="HardcodedText" />
+
+                <com.google.android.material.checkbox.MaterialCheckBox
+                    android:layout_width="wrap_content"
+                    android:layout_height="match_parent"
+                    android:text="@string/text_one"
+                    app:useMaterialThemeColors="false" />
+                <com.google.android.material.checkbox.MaterialCheckBox
+                    android:layout_width="wrap_content"
+                    android:layout_height="match_parent"
+                    android:text="@string/text_two"
+                    app:useMaterialThemeColors="false" />
+                <com.google.android.material.checkbox.MaterialCheckBox
+                    android:layout_width="wrap_content"
+                    android:layout_height="match_parent"
+                    android:text="@string/text_three"
+                    app:useMaterialThemeColors="false" />
+
+                <TextView
+                    android:id="@+id/text_view_span_radio_button"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="Radio Button"
+                    tools:ignore="HardcodedText" />
+
+                <RadioGroup
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:id="@+id/radioGroup">
```

Comment:
Add a paddingBottom.

You should use
`android:paddingBottom="@dimen/margin_large"`